subsys/bluetooth/host/l2cap.c

```diff
@@ -1253,8 +1254,6 @@ static void le_ecred_reconf_req(struct bt_l2cap *l2cap, uint8_t ident,
 
 	BT_DBG("mtu %u mps %u", mtu, mps);
 
-	result = BT_L2CAP_RECONF_SUCCESS;
```

Comment:
Removing this line and doing the initialisation upon declaration doesn't seem to be fixing anything, or be part of the fix. So please move it out to a separate patch. In fact, is it really even a good idea, since with this change the variable gets written to twice for the error cases instead of just once?

Reply:
Sorry, ignore that: I should have read the commit message